docs/index.md

```diff
@@ -186,9 +186,11 @@ Bear in mind that, if your `Ingress-Nginx-Controller-nginx2` is started with the
   ```
 - Now you install the additional instance of the ingress-NGINX controller like this ;
   ```
-  helm --namespace ingress-controller-2 install ingcontroller-2 ingress-nginx/ingress-nginx  \
-  --set controller.ingressClass=ingress-class-2 \
-  --set controller.ingressClassResource.name=ingress-class-2 \
-  --set controller.ingressClassResource.controllerValue= "k8s.io/ingress-controller-2" 
+  helm install ingress-nginx-2 ingress-nginx/ingress-nginx  \
+  --namespace ingress-nginx-2 \
+  --set controller.ingressClassResource.name=nginx-2 \
+  --set controller.ingressClassResource.controllerValue= "k8s.io/ingress-nginx-2" \
+  --set controller.ingressClassResource.enabled=true \
```

Comment:
seems to be default, so we don't really need that here. See: https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L100

Reply:
I am going to keep the line --set controller.ingressClassResource.enabled=true as this is a FAQ